tests/lib/tools/tests.info

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
+
```

Comment:
Please use this explicitly and not on the shebang line
```suggestion
set -eu
```

Reply:
done